ister.py

```diff
@@ -453,10 +453,18 @@ def copy_os_swupd(args, template, target_dir):
     cmd = "swupd verify --install"
     cmd += " --path={0}".format(target_dir)
     cmd += " --manifest={0}".format(template["Version"])
-    if args.contenturl:
-        cmd += " --contenturl={0}".format(args.contenturl)
-    if args.versionurl:
-        cmd += " --versionurl={0}".format(args.versionurl)
+    if args.contenturl or template.get("MirrorURL"):
+        if template.get("MirrorURL"):
```

Comment:
Ah sorry, I missed one thing because of my thinking that we wouldn't end up using the mirrorURL in the template. I think the order should be switched. Use the args if it exists otherwise use the template. The commandline for ister.py should supercede. This is an odd case though.

Reply:
If ister.py is used directly, the template value will never be set anyway - so no harm, and the code will do the right thing here.